packages/@react-spectrum/menu/src/ActionMenu.tsx

```diff
@@ -21,14 +21,16 @@ import More from '@spectrum-icons/workflow/More';
 import React from 'react';
 import {SpectrumActionMenuProps} from '@react-types/menu';
 import {useMessageFormatter} from '@react-aria/i18n';
+import {useSlotProps} from '@react-spectrum/utils';
 
 function ActionMenu<T extends object>(props: SpectrumActionMenuProps<T>, ref: FocusableRef<HTMLButtonElement>) {
   let formatMessage = useMessageFormatter(intlMessages);
   let buttonProps = filterDOMProps(props, {labelable: true});
   if (buttonProps['aria-label'] === undefined) {
     buttonProps['aria-label'] = formatMessage('moreActions');
   }
-  
+  props = useSlotProps(props, 'actionGroup');
```

Comment:
```suggestion
  props = useSlotProps(props, 'actionMenu');
```

or is this even necessary? the `actionButton` slot should get used because it's inside action menu

Reply:
It's probably needed, at least, it is for me, i need to be able to control menu trigger props https://github.com/adobe/react-spectrum/pull/2209/files#diff-799cdd6006a24f6548e088ae765bbcf6b6933907175a2e092a8dc527ea9fe98aR27